tests/fixtures/RTCIceCandidate_end.json

```diff
@@ -1,3 +1,6 @@
 {
-  "candidate": ""
+  "candidate": "",
```

Comment:
Please revert changes to this file and it must be valid to keep backwards compability

Reply:
Done